ext/groonga/rb-grn-table.c

```diff
@@ -184,7 +184,8 @@ rb_grn_table_inspect_content (VALUE self, VALUE inspected)
 }
 
 /*
- * テーブルの中身を人に見やすい文字列で返す。
+ * This function return contents of a table as a string
+ * easy to understand for human.
```

Comment:
How about

`This function return contents of a table as a string.
It's easy to understand for human.`
or
`This function return contents of a table as a pretty-printed string.`